### PR TITLE
Cityhall changes

### PR DESCRIPTION
**Pull request by DatBoiWeep**
- Changed clock-in locations for Judges & Lawyers
- Relocated DMV to be inside new Cityhall MLO
- Relocated blips
- Stopped old courthouse MLO
- Added Door locks & Metal Detectors

** This needs to be pushed after Chang's Cityhall MLO update **

### CODE REVIEW

**Comment by minipunch** [comment]
I know @maxgestic added metal detectors to the old courthouse, did you make sure those were in the new one also @ChangyBoi  ?

**Comment by minipunch** [comment]
Or I guess @DatBoiWeep

**Comment by minipunch** [comment]
Also, do we need to add door locks as we have in the old courthouse ?

**Comment by DatBoiWeep** [comment]
I can add some locks to the courtroom specifically and add some metal detectors

**Comment by DatBoiWeep** [comment]
Finished adding additional stuff :)

File: resources/usa_metal_detector/config.lua [hunk 1]
```diff
@@ -1,29 +1,49 @@
...
+		},
+		{ -- LS Courthouse Left
+			info = { id = 8, sound = { range = 10 } },
+			coords = { x = 245.6187, y = -1079.1475, z = 29.2941, radius = 0.5 },
+			entity = { enable = false, heading = nil }
+		},
 	},
 
-	debug = false -- Prints debug information for trouble shooting.
+	debug = true -- Prints debug information for trouble shooting.
```
Comment (minipunch) [inline resources/usa_metal_detector/config.lua:46]:
```suggestion
	debug = false -- Prints debug information for trouble shooting.
```
??

**Review by minipunch — APPROVED** [review]

**Pull request merged**